resources/systemd/openhab2.service

```diff
@@ -1,5 +1,5 @@
 [Unit]
-Description=openHAB 2 - empowering the smart home
+Description=openHAB 2 - Empowering the smart home
```

Comment:
It seemed un-natural at it was. My intention was to capitalise the S and H in Smart Home (as it is in the docs), but I must have misplaced that.